web/src/components/organisms/Workspace/index.tsx

```diff
@@ -0,0 +1,41 @@
+import ProjectCreationModal from "@reearth-cms/components/molecules/Common/ProjectCreationModal";
+import ProjectList from "@reearth-cms/components/molecules/ProjectList/ProjectList";
+import WorkspaceWrapper from "@reearth-cms/components/molecules/Workspace";
+import Greeting from "@reearth-cms/components/molecules/Workspace/Greeting";
+import WorkspaceHeader from "@reearth-cms/components/molecules/Workspace/WorkspaceHeader";
+
+import useHooks from "./hooks";
+
+export type Props = {
+  onWorkspaceModalOpen?: () => void;
+};
+
+const Workspace: React.FC<Props> = ({ onWorkspaceModalOpen }) => {
```

Comment:
Where is this component being used? Can't seem to find it

Reply:
@KaWaite , it is passed to the page component here: `src/components/pages/Workspace/index.tsx`